### DISCUSSION TITLE
Is there a path to address Ergast issues called out by FastF1?

### DISCUSSION THREAD

**Discussion by borolepratik** [jolpica/jolpica-f1#109] [Q&A] [answered]
Wondering if there is interest in addressing https://github.com/theOehrly/Fast-F1/labels/blocked%3Aergast in this project.

**Comment by theOehrly** [comment, marked as answer]
For now, jolpica-f1 is aiming to achieve feature parity with Ergast. These issues are blocked because the data does not exist in Ergast, though. So addressing them is not a top priority at the moment. 
We do plan to extend and improve the API and potentially provide more data. But this is further down on the roadmap. Maybe some of these issues can be addressed next year. In general, yes we are interested in fixing them.
